test-cases/performance/perf-regression-throughput-1250gb-with-ebs-volume.yaml

```diff
@@ -0,0 +1,52 @@
+test_duration: 1580
```

Comment:
I don't think we need it.
We shall use the same one as disk-and-cahce.
Same test exactly, so we can compare the results between EBS and NVMe.

Reply:
I created and updated like  disk-and-cache. Left this test for a whlie